.github/workflows/ci.yml

```diff
@@ -5,46 +5,54 @@ on:
   pull_request:
 
 env:
-  CONSUL_VERSION: 1.6.10
-
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+    
 jobs:
-
-  # Enforces the consistency of code formatting using `.editorconfig` and the `dotnet-format` tool.
-  check-format:
```

Comment:
Sorry for the last minute change request @mkfl, but I just noticed that this PR is removing the `check-format` job!
Could you please add it back?

Reply:
Done